forge-gui/res/cardsfolder/upcoming/ral_monsoon_mage_ral_leyline_prodigy.txt

```diff
@@ -0,0 +1,35 @@
+Name:Ral, Monsoon Mage
+ManaCost:1 R
+Types:Legendary Creature Human Wizard
+PT:1/3
+S:Mode$ ReduceCost | ValidCard$ Instant,Sorcery | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Instant and sorcery spells you cast cost {1} less to cast.
+T:Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | PlayerTurn$ True | Execute$ TrigFlip | TriggerDescription$ Whenever you cast an instant or sorcery spell during your turn, flip a coin. If you lose the flip, CARDNAME deals 1 damage to you. If you win the flip, you may exile NICKNAME. If you do, return him to the battlefield transformed under his owner's control.
+SVar:TrigFlip:DB$ FlipACoin | WinSubAbility$ DBTransform | LoseSubAbility$ DBDamage
+SVar:DBDamage:DB$ DealDamage | Defined$ You | NumDmg$ 1
+SVar:DBCopy:DB$ CopySpellAbility | Defined$ TriggeredSpellAbility | AILogic$ Always | MayChooseTarget$ True
+SVar:DBTransform:DB$ ChangeZone | Origin$ Battlefield | Destination$ Exile | OptionalDecider$ You | RememberChanged$ True | SubAbility$ DBReturn
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Destination$ Battlefield | Transformed$ True | ForgetOtherRemembered$ True | SubAbility$ DBCleanup | StackDescription$ None
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+AlternateMode:DoubleFaced
+DeckNeeds:Type$Instant|Sorcery
+Oracle:Instant and sorcery spells you cast cost {1} less to cast.\nWhenever you cast an instant or sorcery spell during your turn, flip a coin. If you lose the flip, Ral, Monsoon Mage deals 1 damage to you. If you win the flip, you may exile Ral. If you do, return him to the battlefield transformed under his owner's control.
+
+ALTERNATE
+
+Name:Ral, Leyline Prodigy
+ManaCost:no cost
+Colors:blue,red
+Types:Legendary Planeswalker Ral
+Loyalty:2
+K:etbCounter:LOYALTY:X:no Condition:CARDNAME enters the battlefield with an additional loyalty counter on him for each instant and sorcery spell you've cast this turn.
+A:AB$ Effect | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | StaticAbilities$ RalReduceCost | Duration$ UntilYourNextTurn | SpellDescription$ Until your next turn, instant and sorcery spells you cast cost {1} less to cast.
+SVar:RalReduceCost:Mode$ ReduceCost | EffectZone$ Command | ValidCard$ Instant,Sorcery | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Until your next turn, instant and sorcery spells you cast cost {1} less to cast.
+A:AB$ DealDamage | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target to distribute damage to | NumDmg$ 2 | TargetMin$ 1 | TargetMax$ 2 | DividedAsYouChoose$ 2 | SubAbility$ DBDraw | SpellDescription$ NICKNAME deals 2 damage divided as you choose among one or two targets. Draw a card if you control a blue permanent other than NICKNAME.
```

Comment:
wrong tgts

Reply:
I'm unsure what is wrong here. Ral's -2 is worded exactly the same as Electrolyze, from which this ability borrows code.

Reply:
Battle -- I got it now! Scripting in a post-MOM world feels strange now. Fixing